src/graph/DescribeSchemaExecutor.cpp

```diff
@@ -0,0 +1,79 @@
+/* Copyright (c) 2018 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "DescribeSchemaExecutor.h"
+
+namespace nebula {
+namespace graph {
+
+/*static*/ std::string DescribeSchemaExecutor::value2String(const ::nebula::cpp2::Value& value) {
+    switch (value.getType()) {
+        case ::nebula::cpp2::Value::Type::bool_value: {
+            return std::to_string(value.get_bool_value());
+        }
+        case ::nebula::cpp2::Value::Type::int_value: {
+            return std::to_string(value.get_int_value());
+        }
+        case ::nebula::cpp2::Value::Type::double_value: {
+            return std::to_string(value.get_double_value());
+        }
+        case ::nebula::cpp2::Value::Type::string_value: {
+            return value.get_string_value();
+        }
+        case ::nebula::cpp2::Value::Type::timestamp: {
+            return std::to_string(value.get_timestamp());
+        }
+        case ::nebula::cpp2::Value::Type::__EMPTY__: {
+            return std::string();
+        }
+    }
+    return std::string();
+}
+
+/*static*/ std::vector<cpp2::RowValue> DescribeSchemaExecutor::genRows(
+    const std::vector<::nebula::cpp2::ColumnDef>& cols) {
+    std::vector<cpp2::RowValue> rows;
+    for (auto& item : cols) {
+        std::vector<cpp2::ColumnValue> row;
+        row.resize(6);
+        // Field
+        row[0].set_str(item.name);
+        // Type
+        row[1].set_str(valueTypeToString(item.type));
+        // Null
+        if (item.get_could_null() != nullptr) {
+            row[2].set_str(*item.get_could_null() ? "True" : "False");
+        } else {
+            // For testing checking with one value
+            row[2].set_str(std::string());
+        }
+        // Key
+        if (item.get_key_type() != nullptr) {
+            auto keyType = *item.get_key_type();
+            auto keyTypeName = keyTypeToString(keyType);
+            row[3].set_str(keyTypeName);
+        } else {
+            // For testing checking with one value
+            row[3].set_str(std::string());
+        }
+        // Default
+        if (item.get_default_value() != nullptr) {
+            // covert to characters for the column require same type for testing checking
+            row[4].set_str(value2String(*item.get_default_value()));
+        } else {
+            // For testing checking with one value
+            row[4].set_str(std::string());
```

Comment:
If not exist default, return NULL will be better? because it can't distinguish it's NULL or empty string, now NULL is unsupported, you can add `TO DO`, and return "NULL"?

Reply:
I don't think NULL is better. Because NULL can't mean the default is NULL itself.And the mysql return empty when not specify, I think it's a good choice.